bot/exts/christmas/advent_of_code/_helpers.py

```diff
@@ -171,7 +170,7 @@ def _format_leaderboard(leaderboard: typing.Dict[str, dict]) -> str:
     return "\n".join(leaderboard_lines)
 
 
-async def _leaderboard_request(url: str, board: int, cookies: dict) -> typing.Optional[dict]:
+async def _leaderboard_request(url: str, board: str, cookies: dict) -> dict[str, Any]:
```

Comment:
Is this type supposed to be a string? It seems like an odd change from int to string if the expected type will be an int.

Reply:
It's supposed to be a string: https://github.com/python-discord/sir-lancebot/blob/745cd1d6d3d6227d2a1e82cf25611d76221c40cd/bot/constants.py#L38-L41